### ISSUE TITLE
[Bug]: Search function don't work

### ISSUE THREAD

**Issue by ertup0** [yukieiji/UnityExplorer#16]
### Are you on the latest version of UnityExplorer?

- [X] Yes, I'm on the latest version of UnityExplorer.

### Which release are you using?

BepInEx 5.X Mono

### Which game did this occur on?

Occurs in most games that use Unity 5.1 to 2020.x 

### Describe the issue.

This issue is not present in UnityExplorer 4.9.0 (and fork 4.9.2)
When trying to search a "[Unity] ReflectionTypeLoadException: The classes in the module cannot be loaded." exception occurs and search results are not displayed.

### Relevant log output

```shell
[Message:   BepInEx] BepInEx 5.4.19.0 - BATIM (5/19/2024 2:41:56 PM)
[Info   :   BepInEx] Running under Unity v5.6.5.148225
[Info   :   BepInEx] CLR runtime version: 2.0.50727.1433
[Info   :   BepInEx] Supports SRE: True
[Info   :   BepInEx] System platform: Windows
[Message:   BepInEx] Preloader started
[Info   :   BepInEx] Loaded 1 patcher method from [BepInEx.Preloader 5.4.19.0]
[Info   :   BepInEx] 1 patcher plugin loaded
[Info   :   BepInEx] Patching [UnityEngine] with [BepInEx.Chainloader]
[Message:   BepInEx] Preloader finished
[Warning:  HarmonyX] AccessTools.Property: Could not find property for type UnityEngine.Application and name isBatchMode
[Message:   BepInEx] Chainloader ready
[Message:   BepInEx] Chainloader started
[Info   :   BepInEx] 1 plugins to load
[Info   :   BepInEx] Loading [UnityExplorer 4.12.0]
[Message:UnityExplorer] UnityExplorer 4.12.0 initializing...
[Message:UnityExplorer] [UniverseLib] UniverseLib 1.5.6 initializing...
[Message:UnityExplorer] [UniverseLib] Finished UniverseLib initial setup.
[Warning:  HarmonyX] AccessTools.Method: Could not find method for type UnityEngine.Canvas and name get_renderingDisplaySize and parameters 
[Message:   BepInEx] Chainloader startup complete
[Message:UnityExplorer] [Unity] S13AudioManager(Clone): Loaded 0 sound banks. Total sound banks loaded: 0
[Message:UnityExplorer] [Unity] S13AudioManager(Clone): Loaded 32 mixers: ArchivesRoom, Ballroom, Bridge, CH1 Ambience by Section, CH1Defaults, CH1ScriptedEvents, CH2 Ambience by Section, CH2Defaults, CH3 Ambience by Section, CH3Defaults, CH4 Ambience by Section, CH0 Global Scripted Events, CharacterAnimations, Dialogue, HauntedHouse, Holding, Maintenance, MapRoom, Music, PlayerFoley, PropsInteractive, PropsStatic, ResearchAndDesign, RideStorage, S13BaseMixer, Sewers, SpiralStairs, StartingLift, Vent, Warehouse, TMGAudioMixer, CH3Workshop, 
[Message:UnityExplorer] [Unity] _MasterSoundBank: Initialized sound bank with 20 sounds.
[Message:UnityExplorer] [Unity] Fullscreen: True
Resolution :1920 x 1080
Resolution Index: 8
[Message:UnityExplorer] [Unity] S13AudioManager(Clone): Loaded 0 sound banks. Total sound banks loaded: 0
[Message:UnityExplorer] [UniverseLib] Initialized Legacy Input support
[Message:UnityExplorer] [UniverseLib] Loaded legacy.5.6 bundle for Unity 5.6.5p2
[Message:UnityExplorer] [UniverseLib] UniverseLib 1.5.6 initialized.
[Message:UnityExplorer] Creating UI...
[Message:UnityExplorer] UnityExplorer 4.12.0 (Mono) initialized.
[Message:UnityExplorer] [Unity] Creating instance of singleton component InControlManager
[Message:UnityExplorer] [Unity] S13AudioManager(Clone): Loaded 0 sound banks. Total sound banks loaded: 0
[Message:UnityExplorer] [Unity] S13AudioManager(Clone): InvokeEvent 'evt_game_at_main_menu' with delay 0
[Error  :UnityExplorer] [Unity] ReflectionTypeLoadException: The classes in the module cannot be loaded.
[Error  :UnityExplorer] [Unity] ReflectionTypeLoadException: The classes in the module cannot be loaded.
```

**Comment by ertup0** [comment]
Upon further testing this issue started occuring since unityexplorer 4.9.3
https://github.com/yukieiji/UnityExplorer/releases/tag/v4.9.3

**Comment by yukieiji** [comment]
Sure seems to be an error but it doesn't work? Or is it just a simple error?
The same error occurred in the version before that, it just didn't work, and the fix should make that work, even though the error is still displayed.

**Comment by ertup0** [comment]
The error always occurs when pressing the search button when searching classes and singletons and the search results are not displayed

**Comment by yukieiji** [comment]
fixed in [v4.12.1](https://github.com/yukieiji/UnityExplorer/releases/tag/v4.12.1)
